From eeac00dfe6c49b802f99318c3ff999e2a45d4ebd Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Mon, 28 May 2012 10:27:46 +0800
Subject: [PATCH 1543/2508] ENGR00211132 ESAI: fix build warning

imx-pcm.h should not include imx-ssi.h, or else DRV_NAME is
redefined

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/imx/imx-pcm.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/sound/soc/imx/imx-pcm.h b/sound/soc/imx/imx-pcm.h
index be0d5ff..274af8f 100644
--- a/sound/soc/imx/imx-pcm.h
+++ b/sound/soc/imx/imx-pcm.h
@@ -42,8 +42,6 @@
 
 #include <mach/dma.h>
 
-#include "imx-ssi.h"
-
 struct imx_pcm_runtime_data {
 	int period_bytes;
 	int periods;
-- 
1.7.7.4

